docs/.vitepress/config.mts

```diff
@@ -138,7 +138,12 @@ export default defineConfig({
     },
 
     search: {
-      provider: 'local',
+      provider: 'algolia',
+      options: {
+        appId: 'BWKGTG68ZO',
+        apiKey: '06bc959e3f6ab7eb186cd27653408b04',
```

Comment:
is there risk to having this in the repo? i can throw (a regenerated) key in a Github Actions secret if we don't want this public.

Reply:
It's ok. These are client-side keys, so they are rendered in the html anyways, like google analytics: https://docsearch.algolia.com/docs/docsearch-v3#implementation

Reply:
ah right, i should have know 👍